src/Post/Post.css

```diff
@@ -359,3 +368,41 @@ section.comment-input-section {
   text-decoration: underline;
   cursor: pointer;
 }
+.img-buttons-container {
+  position: relative;
+  top: -50%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 0;
+}
+.next-img-button,
+.prev-img-button {
+  padding: 0;
+  margin: 8px;
+  border-radius: 50%;
+  height: 23px;
+  width: 23px;
+  border: none;
+  background: transparent;
+  box-shadow: 2px 2px 6px 1px rgba(0, 0, 0, 0.28);
+}
+.next-img-button.hidden,
+.prev-img-button.hidden {
+  visibility: hidden;
+}
+.progress-dots-section {
+  display: flex;
+  gap: 5px;
+  justify-content: center;
+  margin-top: 15px;
+}
+.progress-dot {
+  height: 6px;
+  width: 6px;
+  background-color: #8e8e8e;
+  border-radius: 50%;
+}
+.progress-dot.is-blue {
+  background-color: rgb(0, 149, 246);
+}
```

Comment:
When the post is expanded, the progress dot has opacity 0.4, and 0.1 when active. `is-blue` should be is-active or such.